virttest/qemu_devices/qcontainer.py

```diff
@@ -996,7 +996,7 @@ def machine_by_params(self, params=None):
         else:
             pci_bridge_type = 'pci-bridge'
 
-        def machine_q35(cmd=False):
+        def machine_q35(cmd=False, pcie_root_port_params=None):
```

Comment:
This is unnecessary, you could get params directly inside `machine_q35()`

Reply:
No 'params'  in `machine_q35` so i use it in `machine_by_params`.
Am i wrong ?

Reply:
@yanan-fu You should be able to get it directly since `machine_q35()` is defined insie `machine_by_params()`.

Reply:
@vivianQizhu  yes, we can pass `params`  from `machine_by_params()` to `machine_q35()`, and use `pcie_root_port_params` in `machine_q35()`. 
In `machine_q35()`, only the `pcie_root_port_params` is enough now, do we need pass the whole param ?

Reply:
@vivianQizhu  Ah~  Sorry, misunderstand your meaning,  i got the point now, let me update it, thanks!

Reply:
Updated, thanks @vivianQizhu